src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppRebootWatchdog.java

```diff
@@ -49,13 +53,28 @@ public class SCAppRebootWatchdog {
   /** Status indicator if reboot watchdog is serviced */
   private static boolean isRebootWatchdogServiced = false;
 
+  /** The time the watchdog will accept servicing */
+  private static long watchdogReadyTimeMillis;
+
   /**
-   * Enables the reboot watchdog.
+   * Enables the reboot watchdog. The watchdog will be enabled with a default service time of 0.

```

Comment:
Could you please add a comment about the default value of 0. What does 0 mean? Does this mean the watchdog will be serviced immediately after boot? Instead of waiting for after stability?

Reply:
added